howso/direct/client.py

```diff
@@ -2245,7 +2242,12 @@ def react_series(  # noqa: C901
             if isinstance(progress_callback, Callable):
                 progress_callback(progress, response)
 
-        # If the number of series generated is less then requested, raise
+        # put all explanations under the 'explanation' key
+        series = response['series']
+        del response['series']
```

Comment:
Can we drop the `del ...` line here? It will be ineffective at best, and at worst, could delete the contents of `series`.

Reply:
the point is to remove those contents...